tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -1,290 +1,290 @@
-
-            AuthenticationResult authResult = await msalPublicClient
-                    .AcquireTokenByUsernamePassword(s_scopes, labResponse.User.Upn, new NetworkCredential("", labResponse.User.GetOrFetchPassword()).SecurePassword)
-                    .WithCorrelationId(CorrelationId)
-                    .ExecuteAsync(CancellationToken.None)
-                    .ConfigureAwait(false);
-
-            Assert.IsNotNull(authResult);
-            Assert.AreEqual(TokenSource.IdentityProvider, authResult.AuthenticationResultMetadata.TokenSource);
-            Assert.IsNotNull(authResult.AccessToken);
-            Assert.IsNotNull(authResult.IdToken);
-            Assert.IsTrue(string.Equals(labResponse.User.Upn, authResult.Account.Username, StringComparison.InvariantCultureIgnoreCase));
-            AssertTelemetryHeaders(factory, true, labResponse);
-        }
-
-        private async Task RunAcquireTokenWithUsernameIncorrectPasswordAsync(
-            IPublicClientApplication msalPublicClient,
-            string userName)
-        {
-            SecureString incorrectSecurePassword = new SecureString();
-            incorrectSecurePassword.AppendChar('x');
-            incorrectSecurePassword.MakeReadOnly();
-
-            try
-            {
-                var result = await msalPublicClient
-                    .AcquireTokenByUsernamePassword(s_scopes, userName, incorrectSecurePassword)
-                    .WithCorrelationId(CorrelationId)
-                    .ExecuteAsync(CancellationToken.None)
-                    .ConfigureAwait(false);
-            }
-            catch (MsalServiceException ex)
-            {
-                Assert.IsTrue(!string.IsNullOrWhiteSpace(ex.CorrelationId));
-                Assert.AreEqual(400, ex.StatusCode);
-                Assert.AreEqual(InvalidGrantError, ex.ErrorCode);
-                Assert.IsTrue(ex.Message.StartsWith("AADSTS50126"));
-
-                return;
-            }
-
-            Assert.Fail("Bad exception or no exception thrown");
-        }
-
-        private async Task RunHappyPathTestAsync(LabResponse labResponse, string federationMetadata = "")
-        {
-            var factory = new HttpSnifferClientFactory();
-            var msalPublicClient = PublicClientApplicationBuilder
-                .Create(labResponse.App.AppId)
-                .WithTestLogging()
-                .WithHttpClientFactory(factory)
-                .WithAuthority(labResponse.Lab.Authority, "organizations")
-                .Build();
-
+using Microsoft.Identity.Client.TelemetryCore;
+using Microsoft.Identity.Client.Utils;
+using Microsoft.Identity.Test.Common;
+using Microsoft.Identity.Test.Common.Core.Helpers;
+using Microsoft.Identity.Test.Integration.net45.Infrastructure;
+using Microsoft.Identity.Test.LabInfrastructure;
+using Microsoft.Identity.Test.Unit;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Integration.HeadlessTests

```

Comment:
We should not add an integration test for this. Unit tests are sufficient.

Reply:
I did not add a test here. this file was changed by VS for line endings